src/transformers/models/bert/modeling_bert.py

```diff
@@ -193,9 +199,17 @@ def forward(
 
         if position_ids is None:
             position_ids = self.position_ids[:, past_key_values_length : seq_length + past_key_values_length]
+        # Setting the token_type_ids to the registered buffer in constructor where it is all zeros, which usually occurs
+        # when its auto-generated, registered buffer helps users when tracing the model without passing token_type_ids, solves
+        # issue #5664
```

Comment:
Nit, skip a line before the comment, not after (I think it applies to position_ids as is :-) )

Reply:
Thanks @LysandreJik and @sgugger for reviewing the changes. I have addressed the Nit comment.